x/authz/codec/cdc.go

```diff
@@ -6,10 +6,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-var (
-	Amino     = codec.NewLegacyAmino()
-	ModuleCdc = codec.NewAminoCodec(Amino)
-)
+var Amino = codec.NewLegacyAmino()
```

Comment:
Given that this `ModuleCdc` was exported, we need an API breaking changelog for it, saying to import `Amino` instead, given it is now public.
Chains were doing as the SDK and named their amino codec `ModuleCdc` as well (https://github.com/search?type=code&auto_enroll=true&q=%22.ModuleCdc%22+language%3AGo&l=Go).